packages/next/src/server/app-render/app-render.tsx

```diff
@@ -539,7 +541,7 @@ async function getRSCPayload(
     f: [[initialTree, seedData, initialHead]],
     m: missingSlots,
     G: GlobalError,
-  } satisfies RSCPayload & { P: React.ReactNode }
+  } satisfies InitialRSCPayload & { P: React.ReactNode }
```

Comment:
drive-by to remove one layer of indirection. I wonder if we even need `satisfies` here.